lib/db.py

```diff
@@ -0,0 +1,112 @@
+import boto3
+
+# for use imported elsewhere
+dynamo = boto3.resource('dynamodb', endpoint_url="http://localhost:8000")
```

Comment:
This should probably be an environment variable.

Reply:
Done in 441988a